### DISCUSSION TITLE
Support Markdown In Category Descriptions

### DISCUSSION THREAD

**Discussion by parker02311** [Paymenter/Paymenter#244] [Ideas]
Currently only the homepage appears to support markdown and it would be great to have the ability for the categories to support having markdown descriptions.

**Comment by finnjosten** [comment]
Also in product descriptions (last time I checked it didn't work)

**Comment by CorwinDev** [comment]
Suggestion added in 76265e5a588a28c3f726fc944db4710dcee58256
